models/uniswap/bnb/uniswap_bnb_pools.sql

```diff
@@ -0,0 +1,29 @@
+{{ config(
+    tags=['dunesql'],
+    schema = 'uniswap_bnb',
```

Comment:
fyi @Hosuke -- in case i haven't explicitly called this out. when creating a new spell on dunesql, we need to ensure the schema used is already created on spark too -- even if it's a different spell, just need the schema to exist.

i'm viewing on databricks directly, so you'd have to double check on the dune app instead likely, but here is proof for this schema:
![image](https://github.com/duneanalytics/spellbook/assets/102681548/0855c451-025b-40e6-9d71-8f7edcbb7fa5)

this CI step `check for missing legacy models` also tries to help us catch it: 
https://github.com/duneanalytics/spellbook/actions/runs/5800389627/job/15722400771?pr=3763

Reply:
long way of me saying: we should be good here :)

Reply:
So the schema need to be existed in spark legacy sql first, then we can use it in the dune sql?

Reply:
not necessarily first, just in addition to dunesql. it has to do with how `dbt run` works behind-the-scenes, where it'll first run queries on the information schema to search for all schemas in a dbt project. since we share engines in the project, it runs on both for all models, legacy or not. we haven't found a quick, simple way to pass in the `tag:dunesql` type filter to that process like we do elsewhere. this is just our quick workaround fix.